Makefile.in

```diff
@@ -354,17 +354,12 @@ MAN_PAGES :=  \
 install-no-docs:
 	mkdir -p "$(TLIB)" "$(TBIN)" "$(TMAN)"
 	$(CP) "$(LIB)/." "$(TLIB)/"
-	sed "/^lib=/s;.*;lib='$(TLIB)';"			\
-		<"$(BIN)/mlton" >"$(TBIN)/mlton"
+	$(CP) "$(BIN)/." "$(TBIN)/"
 	chmod a+x "$(TBIN)/mlton"
```

Comment:
Since you are doing `$(CP) "$(BIN)/." "$(TBIN)/"`, you don't need lines 358 (`chmod a+x "$(TBIN)/mlton"`) through 366  (`cd "$(BIN)" && $(CP) "$(LEX)$(EXE)" "$(NLFFIGEN)$(EXE)" "$(PROF)$(EXE)" "$(YACC)$(EXE)" "$(TBIN)/"`).  That is, by copying the contents of the `$(BIN)` directory to `$(TBIN)`, you don't need special handling of individual items in `$(BIN)`.